src/Components/CreateImageWizard/formComponents/ReviewStepTables.js

```diff
@@ -93,14 +113,22 @@ export const RepositoriesTable = () => {
             </Tr>
           </Thead>
           <Tbody data-testid="repositories-tbody-review">
-            {repositories.map((repo, repoIndex) => (
-              <Tr key={repoIndex}>
-                <Td className="pf-m-width-60">{repo.baseurl}</Td>
-              </Tr>
-            ))}
+            {repositories.map((repo, repoIndex) => {
```

Comment:
This could be cleaned up a bit. You don't a nested return nor the jsx brackets here:
```javascript
            {repositories.map((repo, repoIndex) => (
              <Tr key={repoIndex}>
                <Td className="pf-m-width-60">
                  <RepoName repoUrl={repo.baseurl} />
                </Td>
              </Tr>
            ))}
```

Reply:
done, thank you @regexowl